docs/remote-docs.sh

```diff
@@ -80,12 +96,12 @@ function pub_pages() {
         $publisher $f
     done
 
-    for c in "container" "image" "pod" "volume" ""; do
-        local cmd=${c:+-$c}
-        for s in $($PODMAN $c --help | sed -n '/^Available Commands:/,/^Flags:/p' | sed -e '1d;$d' -e '/^$/d' | awk '{print $1}'); do
-            $publisher $(echo $source/podman$cmd-$s.*)
-        done
-    done
+    while IFS= read -r cmd; do
+        file="podman-${cmd// /-}"
+        if [ -f $source/$file.* ]; then
```

Comment:
Could you make this `-f $source/$file.1.md` instead? Also in the next line? `*` barfs horribly if there are `*~` or `*.bkp` or other such files.